public/components/layer_config/documents_config/document_layer_source.tsx

```diff
@@ -357,6 +366,17 @@ export const DocumentLayerSource = ({
               compressed
             />
           </EuiFormRow>
+          <EuiFormRow>
+            <EuiCheckbox
+              id={`${selectedLayerConfig.id}-apply-global-filter`}
+              label={i18n.translate('documentLayer.applyGlobalFilters', {
+                defaultMessage: 'Apply global filters',
+              })}
+              checked={applyGlobalFilters}
```

Comment:
We can directly use `selectedLayerConfig.source?.applyGlobalFilter ?? true` here without 
```
const [applyGlobalFilters, setApplyGlobalFilters] = useState<boolean>(
    selectedLayerConfig.source?.applyGlobalFilter ?? true
  );
```

Reply:
Ack